### DISCUSSION TITLE
[Item]_英文期刊论文阅读框架

### DISCUSSION THREAD

**Discussion by qingyiwujia** [windingwind/zotero-better-notes#615] [Note Templates]
**超级简洁的模板，需要搭配更新jcr分区的插件使用**

```
${await new Promise(async (r) => {
    let res = `<h1><font face="微软雅黑" color=#a06000>🍓${topItem.getField("title")}笔记🌴</font></h1>
<h2><font face="微软雅黑" color=#669c7e>🍟文献来源</font></h2>
JCR分区：${topItem.getField("archive")}<br>
中科院大类：${topItem.getField("archiveLocation")}<br>
中科院小类：${topItem.getField("libraryCatalog")}
<br>
影响因子：${topItem.getField("callNumber")}
<br>
5年影响因子：${topItem.getField("rights")}
<h2><font face="微软雅黑" color=#bf0000>🎈解决的问题</h2>
<h2><font face="微软雅黑" color=#bf22ff>🎡核心思路</font></h2>
<h2><font face="微软雅黑" color=#820000>🏀方法</font></h2>
<h2><font face="微软雅黑" color=#c66f00>🔑结论</font></h2>
`;
    r(res);
  })}
```
**预览**
 <h1><font face="微软雅黑" color=#a06000>🍓${topItem.getField("title")}笔记🌴</font></h1>
<h2><font face="微软雅黑" color=#669c7e>🍟文献来源</font></h2>

JCR分区：${topItem.getField("archive")}

中科院大类：${topItem.getField("archiveLocation")}

中科院小类：${topItem.getField("libraryCatalog")}

影响因子：${topItem.getField("callNumber")}

5年影响因子：${topItem.getField("rights")}
<h2><font face="微软雅黑" color=#bf0000>🎈解决的问题</h2>

<h2><font face="微软雅黑" color=#bf22ff>🎡核心思路</font></h2>

<h2><font face="微软雅黑" color=#820000>🏀方法</font></h2>

<h2><font face="微软雅黑" color=#c66f00>🔑结论</font></h2>

**Comment by MaigasNg** [comment]
需要哪个插件呢
